tests/main.cpp

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
Can you add a docstring about the purpose of this file. In particular what does it bring wrt test_morphology.cpp